WORKSPACE

```diff
@@ -82,7 +82,7 @@ http_archive(
 
 http_archive(
     name = "github_nlohmann_json",
-    build_file = "//third_party/json:nlohmann_json.BUILD",
+    build_file = "//bazel:nlohmann_json.BUILD",
```

Comment:
I'd like to reserve `third_party/json` for submodule, moving the bazel files into `/bazel` build and ignoring all of `third_party` directory, which is now used for predictable recursive cloning when using CMake.